SDL/Quintuplet.h

```diff
@@ -85,7 +85,7 @@ namespace SDL {
     Buf<TDev, FPX> phi_buf;
     Buf<TDev, FPX> score_rphisum_buf;
     Buf<TDev, uint8_t> layer_buf;
-    Buf<TDev, bool> isDup_buf;
+    Buf<TDev, char> isDup_buf;
```

Comment:
Why was isDup changed to char? At first glance I don't see why this is required, so I'm just curious.

Reply:
See comment above.